backend/app/controllers/api/v1/admin/reports_controller.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class Api::V1::Admin::ReportsController < Api::V1::Admin::BaseController
+  def learner_activity
+    months_ago = params[:months_ago] || 1
+    csv = LearnerActivityReport.new(months_ago: months_ago)
```

Comment:
this is trifling, but the `csv` variable isn't holding a CSV, it's holding the `report`, so it should be renamed something accordingly, maybe `report` or `lar` like we did below